Automattic/wp-calypso
# Domains: Remove the TLD filter bar from domain suggestions and move the TLD filtering under Filters

We want to simplify the domain suggestion screen and one thing we believe would be useful is to get rid of the TLD Filter bar.

#### Changes proposed in this Pull Request

* Remove the TLD Filter bar from domain suggestions
* Add TLD filter under the Filters popover menu

Before
<img width="1082" alt="Screenshot 2021-05-14 at 17 11 47" src="https://user-images.githubusercontent.com/1355045/118282954-8064f380-b4d7-11eb-82ce-90e9bd636f0c.png">

After
<img width="1074" alt="Screenshot 2021-05-14 at 17 09 05" src="https://user-images.githubusercontent.com/1355045/118282983-878c0180-b4d7-11eb-94d2-624def03f266.png">

Before
<img width="1001" alt="Screenshot 2021-05-14 at 17 12 58" src="https://user-images.githubusercontent.com/1355045/118283104-adb1a180-b4d7-11eb-9665-c88effca2a7b.png">

After
<img width="1002" alt="Screenshot 2021-05-14 at 17 09 21" src="https://user-images.githubusercontent.com/1355045/118283124-b3a78280-b4d7-11eb-8a43-b3b9b56b78ab.png">

After with opened Filter
<img width="1044" alt="Screenshot 2021-05-18 at 11 02 14" src="https://user-images.githubusercontent.com/1355045/118642841-7ea37480-b7e4-11eb-9f0e-5242fd7fb473.png">

#### Testing instructions

* Spin up the PR locally or visit the calypso.local version
* go to the signup domain suggestion screen or the add new domain to this site screen and verify that there is no TLD Filter bar visible
* try to filter suggestions by TLD using the Filter popover menu
